### ISSUE TITLE
About itm loss

### ISSUE THREAD

**Issue by qibao77** [salesforce/LAVIS#266]
Thank you for your code! When I reproduce the stage 1 trainging, I find that the itm loss does not convergent, is it normal? Or is there any trick? (note: I replace the bert with xlmr model)

**Comment by dxli94** [comment]
No trick here. You may try lower learning rate, or use cleaner datasets.

**Comment by chenyzh28** [comment]
> 

When I pretrain BLIP on chinese dataset, I also met this question, the ITM and LM loss do not converge. Have you solve this problem?

**Comment by LiJunnan1992** [comment]
@qibao77 We use customised implementation for the mixture of encoder-decoder (med.py) model. It has a different architecture to that of bert, even though it is initialized from bert weights. If xlmr is used, there needs to be a customised implementation as well.

**Comment by ldfandian** [comment]
> 

can you share please more detail here?

by Chinese bert, do you mean that you change the vocab or something else?
what the new learning rate did you use to make it converge?
